frontend/src/modules/history/components/Translation.js

```diff
@@ -119,6 +124,21 @@ export default class Translation extends React.Component<Props> {
                     />
                 </div>
                 <menu className="toolbar">
+                { translation.rejected ?
+                    // Delete Button
+                    <Localized
+                        id="history-translation-button-delete"
+                        attrs={{ title: true }}
+                    >
+                        <button
+                            className='delete far'
+                            title='Delete'
+                            onClick={ this.delete }
+                        ></button>
```

Comment:
When an element is empty, you can (and should) use an autoclosing tag (`<button ... />`).